the-blue-alliance-ios/View Elements/Events/RankingCellViewModel.swift

```diff
@@ -35,12 +35,46 @@ struct RankingCellViewModel {
 
         teamNumber = String(describing: eventRanking.teamKey!.teamNumber)
         teamName = eventRanking.teamKey!.team?.nickname ?? eventRanking.teamKey!.name
+        
```

Comment:
Not sure if this is outdated but - re: conversation in Slack, shouldn't this code be in `tiebreakerInfoString` ?

Reply:
should be fixed in 531ab95